include/nbl/builtin/hlsl/bxdf/fresnel.hlsl

```diff
@@ -172,58 +172,66 @@ struct ReflectRefract
     using vector_type = vector<T, 3>;
     using scalar_type = T;
 
-    static this_t create(bool r, NBL_CONST_REF_ARG(vector_type) I, NBL_CONST_REF_ARG(vector_type) N, scalar_type NdotI, scalar_type NdotTorR, scalar_type rcpOrientedEta)
+    static this_t create(NBL_CONST_REF_ARG(vector_type) I, NBL_CONST_REF_ARG(vector_type) N, scalar_type NdotI, scalar_type rcpOrientedEta)
     {
         this_t retval;
         retval.I = I;
         retval.N = N;
         retval.NdotI = NdotI;
-        retval.NdotTorR = NdotTorR;
         retval.rcpOrientedEta = rcpOrientedEta;
         return retval;
     }
 
-    static this_t create(bool r, NBL_CONST_REF_ARG(Refract<scalar_type>) refract, scalar_type rcpOrientedEta)
+    static this_t create(NBL_CONST_REF_ARG(Refract<scalar_type>) refract, scalar_type rcpOrientedEta)
     {
         this_t retval;
         retval.I = refract.I;
         retval.N = refract.N;
         retval.NdotI = refract.NdotI;
-        retval.NdotTorR = hlsl::mix(refract.NdotI, refract.NdotT, r);
         retval.rcpOrientedEta = rcpOrientedEta;
         return retval;
     }
```

Comment:
both `create` are useless, one can fill out the struct members without much issue by themselves